python/tvm/relax/transform/op_legalizer.py

```diff
@@ -290,6 +302,20 @@ def _nn_adaptive_max_pool2d(bb: BlockBuilder, args: List[Expr], attrs: Attrs, ou
     )
 
 
+def cross_entropy(x, y):
```

Comment:
Let’s use name `_cross_entropy` since the prepending underscore means this function is used internally.

Reply:
Oh, but I noticed that the similar functions (e.g. line: 126, `expand_dims`) are named without prepending underscore

Reply:
Yes that is because the `expand_dims` is already an internal one.

Reply:
But `_cross_entropy` defined outside looks like a function that returns `bb.call_te` (like `_relu`, `_sum`), will it cause confusions? In this case `cross_entropy` has a prefix `nn`. But if the similar situation happens in a normal op, we need to use another naming rule.

Reply:
Yeah sounds good.